src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -265,6 +265,7 @@
         <Product>$(TestProduct)</Product>
         <Architecture>$(Platform)</Architecture>
         <Configuration>$(ConfigurationGroup)</Configuration>
+        <Configuration Condition="'$(ConfigurationSuffix)' != ''">$(Configuration)-$(ConfigurationSuffix)</Configuration>
```

Comment:
Given that we have a ten character constraint on this value I will go with your suggestion so folks can decide if they want the hyphen or not.